src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -14,8 +14,8 @@
     // Default configuration values
     public static final int DefaultTimeout = 1000;
     public static final String DefaultApiUrlBase  = "https://wovn.global.ssl.fastly.net";
+    public static final String DefaultWidgetVersionUrlPrefix = "https://cdn.wovn.io/widget/";
     public static final String DefaultApiUrlProduction  = DefaultApiUrlBase + "/v0/";
-    public static final String DefaultVersionedWidgetUrlProduction  = DefaultApiUrlBase + "/widget";
     public static final String DefaultApiUrlDevelopment = "http://localhost:3001/v0/";
     public static final String DefaultSnippetUrlProduction  = "//j.wovn.io/1";
```

Comment:
change this to `https://j.wovn.io/1`. I don't want to further encourage usage of `//`